fdbserver/TestTLogServer.actor.cpp

```diff
@@ -183,15 +184,16 @@ ACTOR Future<Void> TLogTestContext::sendPushMessages(TLogTestContext* pTLogTestC
 	TraceEvent("TestTLogServerEnterPush", pTLogTestContext->workerID);
 
 	state uint16_t logID = 0;
-	for (logID = 0; logID < pTLogTestContext->numLogServers; logID++) {
+	for (; logID < pTLogTestContext->numLogServers; logID++) {
 		state Reference<TLogContext> pTLogContext = pTLogTestContext->pTLogContextList[logID];
 		bool tLogReady = wait(pTLogContext->TLogStarted.getFuture());
 		ASSERT_EQ(tLogReady, true);
 	}
 
+	state int i = 0;
```

Comment:
Is this needed?

Reply:
Realized this PR is still draft. Ignore this comment if you plan to undo this change.

Reply:
the code is going through a lot of flux. if you want, you can help review the next set of changes, which will have more configurations to test tLog recovery.